sdk/python/Dockerfile.conformance

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Dockerfile for building the source code of conformance tests
-FROM python:3.10-alpine
+FROM python:3.10
```

Comment:
Do we need to use not alpine image for the Conformance tests ?

Reply:
torch library is not getting installed on alpine image